playground/mocks/data/idp/idx/authenticator-verification-phone-2.json

```diff
@@ -0,0 +1,269 @@
+{
```

Comment:
This is for voice call being primary and sms being secondary option for the code to be sent by.